recipe/meta.yaml

```diff
@@ -19,11 +19,12 @@ source:
 
 
 build:
-  number: 0
+  number: 1
   # Since 1.11, power9 seems to be required.
   skip: true  # [ppc64le]
   skip: true  # [cuda_compiler_version == "10.2" or cuda_compiler_version == "11.0" or cuda_compiler_version == "11.1"]
   skip: true  # [win and cuda_compiler_version == "11.2"]
```

Comment:
As CUDA 11.2 is the new minimum in conda-forge ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/1708 ), we can drop `skip`s for older CUDA versions

```suggestion
```

Reply:
Converted to issue: https://github.com/conda-forge/onnxruntime-feedstock/issues/89